compiler/package.json

```diff
@@ -37,14 +37,18 @@
   "scripts": {
     "dev": "tsc --watch",
     "build": "tsc",
-    "playground": "tsx src/playground.ts"
+    "playground": "tsx src/playground.ts",
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix",
+    "format": "prettier --write ."
   },
   "dependencies": {
     "chevrotain": "^10.1.2",
     "just-clone": "^3.2.1"
   },
   "devDependencies": {
     "@types/node": "^14.14.21",
+    "prettier": "^3.3.3",
     "tslint": "^6.1.3",
```

Comment:
Let's remove tslint as it's not being used. Also can we move prettier to root-level devDependency? That is where the lint packages are and prettier would be useful for all modules